example/themes/original/src/components/Messages/index.tsx

```diff
@@ -122,12 +122,29 @@ const MessagesUI = (props: MessagesParams) => {
 		setMessage && setMessage('')
 	}
 
+	const getLogisticTag = (status: any) => {
+		switch (parseInt(status)) {
```

Comment:
Put this function inside util file and use obj dictionary instead switch statement